app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,15 @@
 <resources>
     <string name="app_name">DemoAppkotlin</string>
+    <string name="jay_shah">Android Ui &amp; All Toasts</string>
+    <string name="android_btn">Android UI</string>
+    <string name="radiobtn_male">Male</string>
+    <string name="radiobtn_female">Female</string>
+    <string name="textview_pref">Your Prefrences</string>
+    <string name="checkbox_kotlin">Kotlin</string>
+    <string name="checkbox_ios">IOS</string>
+    <string name="checkbox_angular">Angular Js</string>
+    <string name="btn_submit">Submit</string>
+    <string name="customToastText">All Done</string>
+    <string name="android_Ui_Button">New Button</string>
+    <string name="Basic_kotlin_gender_textview">Gender:-</string>
```

Comment:
follow same naming convention , underscore not camel case

Reply:
Done